clients/launcher/lib/widgets/welcome_modal.dart

```diff
@@ -110,191 +188,631 @@ class _WelcomeModalContentState extends State<_WelcomeModalContent> {
+                    children: [
+                      const Text(
+                        'BIP39 Information',
+                        style: TextStyle(fontSize: 9, fontWeight: FontWeight.bold, color: Colors.white),
+                      ),
+                      const SizedBox(height: 2),
+                      _buildCompactInfoRow('BIP39 Hex:', _currentWalletData['bip39_hex'] ?? ''),
+                      Row(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          SizedBox(
+                            width: 80,
+                            child: Text(
+                              'BIP39 Bin:',
+                              style: const TextStyle(fontSize: 8, color: Colors.white70),
+                            ),
+                          ),
+                          Expanded(
+                            child: SelectableText.rich(
+                              TextSpan(
+                                children: [
+                                  TextSpan(
+                                    text: _currentWalletData['bip39_bin'] ?? '',
+                                    style: const TextStyle(
+                                      fontFamily: 'RobotoMono',
+                                      fontSize: 8,
+                                      color: Colors.white,
+                                    ),
+                                  ),
+                                  TextSpan(
+                                    text: _currentWalletData['bip39_csum'] ?? '',
+                                    style: const TextStyle(
+                                      fontFamily: 'RobotoMono',
+                                      fontSize: 8,
+                                      color: Color(0xFF4CAF50),
+                                    ),
+                                  ),
+                                  if (_currentWalletData['bip39_csum'] != null)
+                                    TextSpan(
+                                      text: '  Checksum hex: ',
+                                      style: const TextStyle(
+                                        fontFamily: 'RobotoMono',
+                                        fontSize: 8,
+                                        color: Colors.white,
+                                      ),
+                                    ),
+                                  if (_currentWalletData['bip39_csum'] != null)
+                                    TextSpan(
+                                      text: _currentWalletData['bip39_csum_hex'] ?? '',
+                                      style: const TextStyle(
+                                        fontFamily: 'RobotoMono',
+                                        fontSize: 8,
+                                        color: Color(0xFF4CAF50),
+                                      ),
+                                    ),
```

Comment:
Use `SailText.<something>`, not textspan. For example "primary12", has a monospace property you can use to make the font monospaced.

```
  static Widget primary12(
    String label, {
    TextAlign? textAlign,
    bool bold = false,
    bool italic = false,
    Color? color,
    bool monospace = false, // here, set this to true!
    TextOverflow overflow = TextOverflow.ellipsis,
  }) {
```

<img width="418" alt="Screenshot 2025-01-22 at 09 04 03" src="https://github.com/user-attachments/assets/2a6feeb9-b3b7-4751-ab25-31fc710a68b5" />